lib/utils.pm

```diff
@@ -754,6 +754,11 @@ sub power_action {
             record_soft_failure('bsc#1055462');
             $shutdown_timeout *= 3;
         }
+        if (get_var("OFW") && check_var('DISTRI', 'opensuse') && check_var('TEST', 'create_hdd_gnome')) {
+            wait_screen_change(sub { send_key 'ctrl-alt-f2'; }, 1);
```

Comment:
Why do we need to switch to tty2 for this test even we haven't detected an issue? Probably, we should collect logs here. On top of that, let's combine this with workaround above for SLE 15.

Reply:
thank you @rwx788 for the review.
You are right, the switch to tty2 was useless, I captured the log in a manual test using the qcow2 generated by the "create_hdd_gnome" test.
But I did not combine with the workaround of SLE 15 because I do not know what is related problem (I do not have access to bsc#1055462)  I prefered to create a new bug boo#1057637